src/routes/PublicLayoutRouter/PublicLayoutRouter.js

```diff
@@ -181,6 +182,10 @@ export function GroupExplorerLayoutRouter () {
 export function PublicPageHeader () {
   return (
     <div styleName='background'>
+      <Helmet>
+        <title>Hylo: Public</title>
+        <meta name='description' content='Hylo: Public content' />
```

Comment:
I'm assuming other `meta` tags within the `Helmet` component lower down over-write this (e.g. `PostDetails`, etc). Is that true?

Reply:
yes